dockers/docker-fpm/config.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
this maybe a good idea, currently it is not clear when the configuration in the docker should be updated and what should happen in the docker.